src/components/Flow.test.tsx

```diff
@@ -18,7 +18,9 @@ const props: FlowProps = {
     nodeDragging: false,
     onDrag: jest.fn(),
     language,
-    translating: baseLanguage.iso !== language.iso && baseLanguage.name === language.name,
+    translating:
+        baseLanguage.iso !== language.iso &&
+        baseLanguage.name === language.name,
     definition,
```

Comment:
Man, just tons of reformatting like this in this PR. Cool with tweaking stuff you're touching, but if you want to do this much reformatting, it really belongs in it's own PR that only does that.

Reply:
i think most if not all of the reformatting is introduced by [this commit](https://github.com/nyaruka/floweditor/pull/129/commits/ffc00454b35fd6b7783b88713f9739b42fc8c4bc), happy to revert.